griptape/drivers/embedding/sagemaker_embedding_driver.py

```diff
@@ -0,0 +1,36 @@
+import json
+import boto3
+from typing import Any
+from attr import define, field, Factory
+from griptape.drivers import BaseEmbeddingDriver
+
+@define
+class AmazonSagemakerEmbeddingDriver(BaseEmbeddingDriver):
+
+  endpoint: str = field(kw_only=True)
+  session: boto3.Session = field(
+    default = Factory(lambda: boto3.Session()),
+    kw_only = True
+  )
+  sagemaker_client: Any = field(
+    default = Factory(lambda self: self.session.client("runtime.sagemaker"), takes_self=True),
+    kw_only = True
+  )
+
+  def try_embed_string(self, string: str) -> list[float]:
+
+    # text_inputs can take a list of string and generate a list of embeddings
+    # the length of the embedding key in the response will be the same as the text_inputs array
+    payload = {
+      "text_inputs": string
+    }
```

Comment:
Which models have you tested this with? Do all models take this standard payload?

We've noticed there is quite a bit of variability across payloads in SageMaker Jumpstart, at least in Prompt Drivers.

Reply:
I created an endpoint on sagemaker with `Bloom 7B1 Embedding` and tested the flow. I initially got the error that only normalize and text_inputs are valid payload keys

Reply:
Just tried it with `GPT-J 6B Embedding`, its the same request format